stylesheets/portfolio.css

```diff
@@ -0,0 +1,66 @@
+body {
+    display: flex;
+    flex-direction: column;
+    background-color: #01395e;
+    color: #ffffff;
+    margin: 0px;
+}
+
+header {
+    display: flex;
+    padding: 10px 100px;
+    margin: 0px 25px;
+}
+
+h1 {
+    margin-right: auto;
+}
+
+ul {
+    display: flex;
+    gap: 20px;
+}
+
+li {
+    list-style-type: none;
+}
+
+a {
+    text-decoration: none;
+    color: #e0c389;
+}
+
+main {
```

Comment:
on your portfolio page your footer is overlapping your content. One way to fix this is by adding a height to your main tag. I tested it with `height: 500px;` and it worked. You can play around with the height value to get your desired look.